lumen/app/Jobs/ImportEvent.php

```diff
@@ -27,28 +27,26 @@ public function __construct($event)
     public function handle(MongoClient $client)
     {
         $eventStorage = $client->test->events;
-
         $toImport = $this->event;
-
         $event = $eventStorage->findOne([
             'external_id' => data_get($toImport, 'external_id')
-        ]); 
+        ]);
 
-        if (empty($event)) {
+        if(empty($event)) {
             $venueStorage = $client->test->venues;
             $venueId = data_get($toImport, 'metadata.venue_id');
-            
+
             $venue = $venueStorage->findOne([
-                'external_id' => $venueId
+                'external_id' => data_get($toImport, 'metadata.venue_id')
```

Comment:
The variable `$venueId` was declared before, we should reuse it instead of use the `data_get` helper twice.